<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wav_Processor</class>
 <widget class="QMainWindow" name="Wav_Processor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wav_Processor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="FilesLayoutWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>251</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="LoadButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Load File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="DockedFiles">
       <property name="selectionRectVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="OpenButton">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RemoveButton">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalWidget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>261</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QFrame" name="frame_Track">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="Track">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>253</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Track/Song title: </string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_Artist">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="ArtistName">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>253</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Artist name: </string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_AlbumName">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="TrackName">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>253</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Album Name: </string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_Genre">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="Genre">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>253</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Genre:</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_ReleaseYear">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="ReleaseYear">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>253</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Release Year: </string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_TrackNumber">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="TrackNumber">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>253</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Track Number:</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_Comments">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="Comments">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>253</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Comments: </string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PlayButton">
       <property name="text">
        <string>Play File</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="FileButton">
       <property name="text">
        <string>Files</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="EditButton">
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>291</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="2">
      <widget class="QTextEdit" name="textEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Amount = </string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QTextEdit" name="textEdit_4"/>
     </item>
     <item row="2" column="2">
      <widget class="QTextEdit" name="textEdit_2"/>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="pushButton_6">
       <property name="text">
        <string>Noise Gate</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>Normalize</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="pushButton_7">
       <property name="text">
        <string>Echo</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>614</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
